eva_sub_cli/validators/validation_results_parsers.py

```diff
@@ -134,6 +134,10 @@ def clean_read(ifile):
                 if line.startswith('Validation failed with following error(s):'):
                     collect = True
             else:
+                while line and not line.startswith('/'):
+                    # Sometimes there are multiple (possibly redundant) errors listed under a single property,
+                    # we only report the first
```

Comment:
We probably should report that to `biovalidator.js`

Reply:
Good point, I'll make an issue for them.